### PR TITLE
hocr-check: Replace assert calls with basic TAP ok test

### PR DESCRIPTION
**Pull request by kba**
Here's a really simplistic approach on how hocr-check can be more useful.

* test function 'test_ok(boolean, message)' instead of assert
* Failing tests will write 'not ok' and the message
* Passing tests will write 'ok' and the message

### CODE REVIEW

File: hocr-check [hunk 1]
```diff
@@ -80,26 +88,26 @@ doc = html.fromstring(stream.read())
...
 # check for presence of page
-assert doc.xpath("//*[@class='ocr_page']")!=[]
+test_ok(doc.xpath("//*[@class='ocr_page']")!=[], "has a page")
 
 # check that lines are inside pages
 lines = doc.xpath("//*[@class='ocr_line']")
-for line in lines:
-    assert line.xpath("//*[@class='ocr_page']")
+for line_idx, line in enumerate(lines):
+    test_ok(line.xpath("//*[@class='ocr_page']"), "ocr_line %2d in a ocr_page"%(line_idx))
```
Comment (zuphilip) [inline hocr-check]: Isn't this checking for a `ocr_page` inside the `line` element?

Comment (kba) [inline hocr-check]: They are supposed to check for the opposite but don't and should be fixed. All those tests in for loops.

Comment (zuphilip) [inline hocr-check]: Okay, and the current tests are just checking that it is possible to execute `hocr-check` with the correct parameters. I guess that the current output of the command is showing `not okay` lines. Correct?

Comment (kba) [inline hocr-check]: `//elem` without context starts at root, so will output `ok`.

**Comment by kba** [comment]
Thanks for merging, you can see the output at http://digi.bib.uni-mannheim.de/ocr-fileformat/#validate

**Pull request merged**